src/link-graph.spec.js

```diff
@@ -0,0 +1,9 @@
+import test from 'tape';
+import React from 'react';
+import LinkNodesExample from './example/link-graph-simple.example';
```

Comment:
I use `.example` to denote example code, and `.spec` to denote tests. If we merge these into the main react-vis codebase I can change the convention.